card/mod/search/set/self/search.rb

```diff
@@ -1,5 +1,3 @@
-include_set Abstract::SearchParams
```

Comment:
Are you sure that you want to remove that?
I see methods from that abstract set called in this file, e.g. query_params

Reply:
:search is a Search card.  Search has Abstract::Search.  Abstract::Search has Abstract::SearchParams.  So we're good. (it's there, I checked.)